### PR TITLE
Divyanshu | Test | Version 3 - Get all voucher page - Bugs 

### PR DESCRIPTION
**Pull request by dvCodeWorld**
https://app.clickup.com/t/86cvfh0d1

### CODE REVIEW

File: apps/web-giddh/src/app/vouchers/adjust-payment-dialog/adjust-payment-dialog.component.html [hunk 1]
```diff
@@ -23,7 +23,6 @@
                                 [type]="'text'"
                                 [name]="'customerName'"
                                 [label]="localeData?.customer_name"
-                                [cssClass]="'form-control mat-field-border'"
```
Comment (dilpreetsinghofficial) [inline apps/web-giddh/src/app/vouchers/adjust-payment-dialog/adjust-payment-dialog.component.html:26]: why cssClass removing ?

File: apps/web-giddh/src/app/vouchers/advance-search/advance-search.component.ts [hunk 2]
```diff
@@ -336,7 +336,9 @@ export class AdvanceSearchComponent implements OnInit, OnDestroy {
             });
         };
 
-        allDateControlName.forEach(controlName => {
+        // Take one control from range date i.e from and to
+        const allDateFromControlName: string[] = ['voucherDate', 'dueDate', 'expireFrom', 'dueFrom'];
```
Comment (dilpreetsinghofficial) [inline apps/web-giddh/src/app/vouchers/advance-search/advance-search.component.ts]: check 
```suggestion
        const allDateFormControlName: string[] = ['voucherDate', 'dueDate', 'expireFrom', 'dueFrom'];
```

File: apps/web-giddh/src/app/vouchers/list/list.component.ts [hunk 3]
```diff
@@ -281,6 +279,8 @@ export class VoucherListComponent implements OnInit, OnDestroy {
     private currentVoucher: any = null;
     /** Holds Advance Search Filter Temp Keys to show label on filter dialog */
     private advanceSearchTempKeyObj: any = {};
+    /** Holds Currently used Voucher */
+    public activeSearchField: any = null;
```
Comment (dilpreetsinghofficial) [inline apps/web-giddh/src/app/vouchers/list/list.component.ts:283]: correct documentation

File: apps/web-giddh/src/app/vouchers/list/list.component.ts [hunk 4]
```diff
@@ -523,6 +522,7 @@ export class VoucherListComponent implements OnInit, OnDestroy {
         });
 
         this.voucherNumberInput.valueChanges.pipe(debounceTime(700), distinctUntilChanged(), takeUntil(this.destroyed$)).subscribe(search => {
+
```
Comment (dilpreetsinghofficial) [inline apps/web-giddh/src/app/vouchers/list/list.component.ts]: remove extra space

**Review by dilpreetsinghofficial — CHANGES_REQUESTED** [review]

File: apps/web-giddh/src/app/vouchers/adjust-payment-dialog/adjust-payment-dialog.component.html (hunk 1 above)
Comment (dvCodeWorld) [reply apps/web-giddh/src/app/vouchers/adjust-payment-dialog/adjust-payment-dialog.component.html:26]: Not use in material input field

File: apps/web-giddh/src/app/vouchers/advance-search/advance-search.component.ts (hunk 2 above)
Comment (dvCodeWorld) [reply apps/web-giddh/src/app/vouchers/advance-search/advance-search.component.ts]: Done

File: apps/web-giddh/src/app/vouchers/list/list.component.ts (hunk 3 above)
Comment (dvCodeWorld) [reply apps/web-giddh/src/app/vouchers/list/list.component.ts:283]: Done

File: apps/web-giddh/src/app/vouchers/list/list.component.ts (hunk 4 above)
Comment (dvCodeWorld) [reply apps/web-giddh/src/app/vouchers/list/list.component.ts]: Done

**Review by dilpreetsinghofficial — APPROVED** [review]

**Pull request merged**